charts/enterprise-metrics/values.yaml

```diff
@@ -56,7 +56,9 @@ tokengenJob:
   annotations: {}
 
 config:
-  auth_enabled: false
```

Comment:
You probably already knew the default was `true` and so it didn't need to be explicitly included but I thought I'd just mention it anyway as a point of note -- both `auth_enabled: true` and `auth.type: enterprise` are necessary to enable enterprise authentication.